Automattic/jetpack
# Migration plugin: move the way we load tracking assets to the bottom of the function

## Proposed changes:

* When you install the plugin and activate it from the plugin directory and gets redirected to the MtW plugin landing page, the tracking asset is not loaded. This PR moves the tracking assets below the initialization of the connection package.

### Other information:


## Jetpack product discussion


See the issue above
## Does this pull request change what data or activity we track or use?


N/A
## Testing instructions:






* Spin up a JN site with [this branch](https://jurassic.ninja/create/?jetpack-beta&branches.wpcom-migration=fix/tracking-order&wp-debug-log)
* Heading to the MtW landing page
* Use Tracks Vigilante or open developer tool and in console, type: `localStorage.setItem( 'debug', 'dops:analytics' );`
* Click `Get started` CTA
* Check if `jetpack_migration_get_started_click` gets recorded.
* Compare with the version that's on the master branch, you'll see the version on the master won't be recorded the first time.
